tests/unit/db/database_tests.py

```diff
@@ -295,10 +295,55 @@ def test_retrieve_raw_view_results(self):
         self.db.create_document(data)
         self.populate_db_with_documents()
 
-        raw_rslt = self.db.get_view_raw_result('_design/ddoc01', 'view01')
+        raw_rslt = self.db.get_view_result('_design/ddoc01', 'view01', True)
```

Comment:
Use `raw_result=True`.  This will help others in the future to better understand what `True` is referring to when looking at the test and also ensures that the code we suggest in the example where explicitly use `raw_result=True` is correct.